framework/scripts/wazuh_server.py

```diff
@@ -243,6 +257,7 @@ async def worker_main(args: argparse.Namespace, server_config: ServerConfig, log
     from wazuh.core.cluster import local_server, worker
 
     context_tag.set('Worker')
+    start_unix_server('Worker')
```

Comment:
```suggestion
    tag = 'Worker'
    context_tag.set(tag)
    start_unix_server(tag)
```

Reply:
It was changed in edcb389a839c82846356d2ca9e981f422df1c442.